go.mod

```diff
@@ -74,3 +74,5 @@ require (
 	google.golang.org/protobuf v1.32.0 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 )
+
+replace github.com/heimweh/go-pagerduty => github.com/imjaroiswebdev/go-pagerduty v0.0.0-20240205220520-6f8c0393b684
```

Comment:
I know this and all the references to a fork of `github.com/heimweh/go-pagerduty` will be removed but just leaving a comment as a reminder.